packages/amesos2/src/Amesos2_KLU2_def.hpp

```diff
@@ -552,9 +519,12 @@ KLU2<Matrix,Vector>::loadA_impl(EPhase current_phase)
 
   // Only the root image needs storage allocated
   if( this->root_ ){
-    nzvals_.resize(this->globalNumNonZeros_);
-    rowind_.resize(this->globalNumNonZeros_);
-    colptr_.resize(this->globalNumCols_ + 1);
+    host_nzvals_view_ = host_value_type_array(
+      Kokkos::ViewAllocateWithoutInitializing("nzvals"), this->globalNumNonZeros_);
+    host_rows_view_ = host_ordinal_type_array(
+      Kokkos::ViewAllocateWithoutInitializing("rowptr"), this->globalNumNonZeros_);
+    host_col_ptr_view_ = host_ordinal_type_array(
```

Comment:
Minor comment, looks like View label convention is inconsistent with the variable name (e.g. `host_col_ptr_view_` with label `colind`, similar above for rows and `rowptrs`

Reply:
Thanks. I will fix in the upcoming Basker PR unless auto test fails here.